src/test/spec/unified_runner/mod.rs

```diff
@@ -101,30 +101,71 @@ impl IntoFuture for RunUnifiedTestsAction {
     }
 }
 
-#[cfg_attr(feature = "tokio-runtime", tokio::test(flavor = "multi_thread"))]
-#[cfg_attr(feature = "async-std-runtime", async_std::test)]
-async fn test_examples() {
-    let _guard: RwLockWriteGuard<_> = LOCK.run_exclusively().await;
-    run_unified_tests(&["unified-test-format", "examples"]).await;
-}
-
 #[cfg_attr(feature = "tokio-runtime", tokio::test(flavor = "multi_thread"))]
 #[cfg_attr(feature = "async-std-runtime", async_std::test)]
 async fn valid_pass() {
     let _guard: RwLockWriteGuard<_> = LOCK.run_exclusively().await;
-    run_unified_tests(&["unified-test-format", "valid-pass"]).await;
+    #[cfg(feature = "in-use-encryption-unstable")]
+    let skipped_files = &[
+        // TODO RUST-1570: unskip this file (ditto below)
+        "collectionData-createOptions.json",
+        // TODO RUST-1405: unskip this file (ditto below)
+        "expectedError-errorResponse.json",
+        // TODO RUST-582: unskip these files (ditto below)
+        "entity-cursor-iterateOnce.json",
+        "matches-lte-operator.json",
+        // TODO: unskip this file when the convenient transactions API tests are converted to the
+        // unified format (ditto below)
+        "poc-transactions-convenient-api.json",
+    ];
+    #[cfg(not(feature = "in-use-encryption-unstable"))]
+    let skipped_files = &[
+        "collectionData-createOptions.json",
+        "expectedError-errorResponse.json",
+        "entity-cursor-iterateOnce.json",
+        "matches-lte-operator.json",
+        "poc-transactions-convenient-api.json",
+        "kmsProviders-placeholder_kms_credentials.json",
```

Comment:
Is there a ticket for these?

Reply:
the test runner code needed to deserialize these is gated behind in-use-encryption-unstable, so they're skipped when that flag isn't on. i'll add a comment explaining as much